packages/db/package.json

```
@@ -29,7 +29,7 @@
     "@blobscan/logger": "workspace:^0.1.1",
     "@blobscan/open-telemetry": "workspace:^0.0.8",
     "@prisma/client": "^5.5.2",
-    "prisma": "5.5.2"
+    "prisma": "5.17.0"
```

Why not use `^` operator so it installs latest minor update for both `prisma` and `@prisma/client` packages?

```suggestion
    "prisma": "^5.5.2"
```

---

Yes that looks better. I took the opportunity to bump them to the latest version available